drivers/interrupt_controller/exti_stm32.c

```diff
@@ -113,56 +113,56 @@ void stm32_exti_enable(int line)
 	}
 #elif CONFIG_SOC_SERIES_STM32F1X
```

Comment:
How about merge EXTI15..0 for F1, F3, F4? eg:
```
#if CONFIG_SOC_SERIES_STM32F1X || CONFIG_SOC_SERIES_STM32F3X || CONFIG_SOC_SERIES_STM32F4X
if (line >= 5 && line <= 9) {
    irqnum = EXTI9_5_IRQn;
} else if 
```

Reply:
Let's get it in tree, then we'll be able to proceed with optimisations.